data-files/samples/targets.Experiment.Any

```diff
@@ -0,0 +1,129 @@
+{ 
+    description = "targets"; 
+    frameRate = 120; 
+    showHUD = true;
+    renderWeaponStatus = true;
+    cooldownMode = "ring";
+    cooldownColor = Color4(0.7,1.0,0.9,0.75);
+            
+    weapon = { 
+        id = "1hit"; 
+        firePeriod = 0.3;
+        damagePerSecond = 4;
+        autoFire = false;
+    }; 
+    sessions = ( 
+        { 
+            description = "1 target"; 
+            id = "1target"; 
+            settingsVersion = 1; 
+            trials = ( 
+                { 
+                    count = 20; 
+                    ids = ( "static"); 
+                } 
+            ); 
+            
+        }, 
+        { 
+            description = "2 targets"; 
+            id = "2targets"; 
+            settingsVersion = 1; 
+            trials = ( 
+                { 
+                    count = 20; 
+                    ids = ( "static", "static"); 
+                } 
+            ); 
+            
+        }, 
+        { 
+            description = "5 targets"; 
+            id = "5targets"; 
+            settingsVersion = 1; 
+            trials = ( 
+                { 
+                    count = 20; 
+                    ids = ( "static", "static", "static", "static", "static"); 
+                } 
+            ); 
+            
+        }, 
+        { 
+            description = "moving"; 
+            id = "moving"; 
+            settingsVersion = 1; 
+            trials = ( 
+                { 
+                    count = 20; 
+                    ids = ( "moving"); 
+                } 
+            ); 
+            
+        }, 
+        { 
+            description = "jumping"; 
+            id = "jumping"; 
+            settingsVersion = 1; 
+            trials = ( 
+                { 
+                    count = 20; 
+                    ids = ( "jumping"); 
+                } ); 
+            
+        }, 
+        { 
+            description = "mixed types"; 
+            id = "mixedtypes"; 
+            settingsVersion = 1; 
+            trials = ( 
+                { 
+                    count = 7; 
+                    ids = ( "static");
+                }, 
+                { 
+                    count = 7; 
+                    ids = ( "moving");
+                }, 
+                { 
+                    count = 6; 
+                    ids = ( "jumping");
+                }, 
+            ); 
+            
+        }, 
+    ); 
+    
+    settingsVersion = 1; 
+    targets = ( 
+        { 
+            destSpace = "player"; 
+            id = "static"; 
```

Comment:
I usually put the ID at the top of the target definition to help make things clear. Fine to reorder to show you can if that's the goal here.